advanced/dapps/tic-tac-toe/src/app/api/signer/route.ts

```diff
@@ -0,0 +1,19 @@
+import { NextResponse } from "next/server";
+import { privateKeyToAccount } from "viem/accounts";
+
+export function GET() {
+  try {
+    const APPLICATION_PRIVATE_KEY = process.env
+      .NEXT_PUBLIC_APPLICATION_PRIVATE_KEY as `0x${string}`;
```

Comment:
why is this `NEXT_PUBLIC_APPLICATION_PRIVATE_KEY` if its used from an api route?

Reply:
the route is getting the private key from .env.  This route is for getting the dapp signer. 
I didn't get the concern here?

Reply:
NEXT_PUBLIC prefixed env is for bowser. You can check the next documentation [here](https://nextjs.org/docs/pages/building-your-application/configuring/environment-variables)

Reply:
Okay, Understood.